test/newrelic_prov.ts

```diff
@@ -0,0 +1,38 @@
+const Seneca = require('seneca');
+import NewrelicProvider from '../src/newrelic';
+
+const sleep = (millis: any) => new Promise(r=>setTimeout(r,millis))
+
+const s01 = Seneca()
+    .test()
+    .use('promisify')
+    .use(NewrelicProvider, {
+        tracing: {
+            ENABLED: true,
+            ACCOUNT_API_KEY: 'YOU API KEY',
+            SERVICE_NAME: 'ORDU_TRACING_TEST',
+            HOST: 'LOCALHOST',
```

Comment:
Because this is not something that will be easy to change in the future, I would like to point this one out - can we use camelCase for the options? camelCase is more conventional in situations like this.

For example:
```javascript
Seneca()
  .use(NewrelicProvider, {
    tracing: {
      isEnabled: true,
      apiKey: 'YOU_API_KEY',
      serviceName: 'ORDU_TRACING_TEST',
      host: 'LOCALHOST'
    }
  })
```